kahuna/public/stylesheets/main.css

```diff
@@ -1477,15 +1526,16 @@ FIXME: what to do with touch devices
 }
 
 .image-crop {
-    border: solid 5px transparent;
-    max-width: 140px;
-    margin-right: 5px;
+    margin-bottom: 10px;
+    min-width: 100%;
     box-sizing: border-box;
+    background-color: #333;
 }
+
 .image-crop--selected,
 .result--selected,
 .result--selected:hover {
-    border-color: #00adee;
+    border: solid 2px #00adee;
```

Comment:
We should probably hve something like `preview--selected` and `crop--selected` to stop side effects.